src/Components/Footer/index.js

```diff
@@ -1,7 +1,91 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './index.scss';
 
 export default class Footer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      arrowIconOn: false,
+    };
+  }
+
+  handleArrowIcon = () => {
+    this.setState({
+      arrowIconOn: !this.state.ArrowIconOn,
+    });
+  };
+
   render() {
-    return <div></div>;
+    const {arrowIconOn} = this.state;
+    return (
```

Comment:
```suggestion
    const { arrowIconOn } = this.state;
    
    return (
```
- 중괄호 좌우로 한칸 띄워주세요
- 구분되는 로직 끼리른 한줄 띄워 주시는게 좋습니다

Reply:
수정 완료입니다!